### DISCUSSION TITLE
Unable to get 255 bytes in LoRa

### DISCUSSION THREAD

**Discussion by m4mans** [jgromes/RadioLib#1311] [Q&A]
Hi, I am trying to send data over LoRa from LLCC68 one end to SX1262 on other end using esp32 on both sides and I'm using interrupt modes. I am using these configurations on both end:

```
freq = 915 Mhz
bw = 125
sf = 9
cr = 7
syncWord 0x12 
power 10
preambleLength 8

radio.begin(915.0, 125.0, 9, 7, 0x12, 10, 8, 0,false);


```
Issue I'm facing is that at max I'm able to send almost 100 bytes in one transaction. At times I need longer packets, what configurations do I need to change to get longer packets.

**Comment by m4mans** [comment]
@jgromes @StevenCellist Any help would be appreciated thanks.

**Reply by StevenCellist** [reply]
The fact that you haven't gotten any yet is simple: I'm merely a LoRaWAN implementer (pinging me assumes you've checked what I know about, which definitely isn't radio modules); and Mr. Gromes the Awesome has other businesses to attend and is unavailable. I suggest you do some research / practice yourself or practice some patience otherwise :)

**Comment by jgromes** [comment]
I'm going to need some more information about this. What is the behavior you are seeing when trying to send more than 100 bytes? An error code? Something else?
